packages/teams-js/src/public/video.ts

```diff
@@ -217,6 +222,9 @@ export namespace video {
     if (!parameters.videoFrameHandler || !parameters.videoBufferHandler) {
       throw new Error('Both videoFrameHandler and videoBufferHandler must be provided');
     }
+    registerHandler('video.setFrameProcessTimeLimit', (timeLimit: number) =>
+      videoPerformanceMonitor?.setFrameProcessTimeLimit(timeLimit),
```

Comment:
Question for you to think about: I don't know what the performance impact of gathering these statistics are on your video processing. Do you think it would be helpful to add a parameter to these functions that let app developers opt out of statistics gathering if they didn't want it for whatever reason?

Reply:
The client wants it anyway so we won't provide option to opt out